content/influxdb/v2.0/write-data/troubleshoot.md

```diff
@@ -0,0 +1,74 @@
+Common failure scenarios that return an HTTP `4xx` or `5xx` error status code include the following:
+
+- The request exceeded a rate limit.
+- The API token was invalid.
+- The client or server reached a timeout threshold.
+- The size of the data payload was too large.
+- The data was not formatted correctly.
+
+Writes may fail partially or completely even though InfluxDB returns an HTTP `2xx` status code for a valid request. For example, a partial write may occur when InfluxDB writes all points that conform to the bucket schema, but rejects points that have the wrong data type in a field.
+
+## HTTP status codes
+
+InfluxDB uses conventional HTTP status codes to indicate the success or failure of a request.
+Write requests return the following status codes:
+
+- `204` Success. The sent data was correctly formatted and accepted for writing to the bucket. `204` doesn't indicate a successful write operation since writes are asynchronous. See how to [check for rejected points](#review-rejected-points).
+- `400` Bad request. The error indicates the line protocol payload was malformed. The response body contains the first malformed line in the data. All payload data was rejected and not written.
+- `401` Unauthorized. The error may indicate one of the following:
+  - The Authorization: Token header is missing or malformed.
+  - The API token value is missing from the header.
+  - The token does not have sufficient permissions to write to this organization and bucket.
+- `404` Not found. A requested resource was not found. The response body contains the requested resource type, e.g. organization name or bucket, and name.
+- `413` Request entity too large. All payload data was rejected and not written.
+- `429` Too many requests. The token is temporarily over quota. The Retry-After header describes when to try the write again.
+- `500` Internal server error. The default HTTP status for an error.
+- `503` Service unavailable. Server is temporarily unavailable to accept writes. The Retry-After header describes when to try the write again.
+
+The `message` property of the response body may contain additional details about the error.
+
+## Troubleshoot failures
+
+If you notice data is missing in your bucket, check the following:
+- Does the `message` property in the response body contain details about the error?
+- Do all lines contain valid syntax, e.g. line protocol or CSV?
+- Do the data types match? For example, did you attempt to write `string` data to an `int` field?
+- Do the timestamps match the precision parameter?
+
+### Review rejected points
+
+If some of your data did not write to the bucket, check for rejected data points. InfluxDB may have rejected points even if the HTTP request returned "Success".
+To get a log of rejected data points, query the [`rejected_points` measurement](/{{% latest "influxdb" %}}/reference/internals/system-buckets/#_monitoring-bucket-schema) in your organization's `_monitoring` bucket.
```

Comment:
```suggestion
To get a log of rejected data points, query the [`rejected_points` measurement](/influxdb/v2.0/reference/internals/system-buckets/#_monitoring-bucket-schema) in your organization's `_monitoring` bucket.
```

Reply:
I obviously don't understand when not to use this.